tests/eclient/testdata/dev_local_info.txt

```diff
@@ -0,0 +1,241 @@
+
+# Deploy local-manager
+eden pod deploy -n local-manager --memory=512MB docker://lfedge/eden-eclient:afaa332 -p {{template "mngr_port"}}:22 --networks={{template "network"}}
+test eden.app.test -test.v -timewait 10m RUNNING local-manager
+
+# Wait for ssh access
+exec -t 5m bash wait-ssh.sh {{template "mngr_port"}}
+
+# Start local manager application
+exec -t 1m bash local-manager-start.sh
+
+# Obtain local-manager IP address
+exec -t 2m bash get-app-ip.sh local-manager
+source .env
+
+# Configure local server
+eden controller edge-node update --device profile_server_token={{template "token"}}
+eden controller edge-node update --device local_profile_server=$app_ip:8888
+
+# STEP 1: Wait for devinfo status
+exec -t 1m bash get-devinfo-status.sh
+stdout 'ONLINE'
+! stderr .
+
+# STEP 2: Deploy the second app
+eden pod deploy -n app1 --memory=512MB docker://lfedge/eden-eclient:9455582 -p {{template "app_port"}}:22 --networks={{template "network"}}
+test eden.app.test -test.v -timewait 10m RUNNING app1
+
+# Wait for ssh access
+exec -t 5m bash wait-ssh.sh {{template "app_port"}}
+
+# STEP 3: Wait for new devinfo status
+exec sleep 30
+exec -t 1m bash get-devinfo-status.sh
+stdout 'ONLINE'
+! stderr .
+
+# STEP 4: Request for shutdown. Check that app1 stops before local_manager
+exec -t 1m bash put-devinfo-cmd.sh COMMAND_SHUTDOWN &
+exec -t 5m bash wait-for-dev-state.sh SHUTTING_DOWN
+! stderr .
+# Could it already have HALTED? Match HALTING or HALTED
+exec -t 5m bash wait-for-app-state.sh app1 HALT
+! stderr .
+exec -t 1m bash get-appinfo-status.sh local-manager
+! stdout 'RUNNING'
+! stderr .
+
+# Could it already have HALTED? Match HALTING or HALTED
+exec -t 5m bash wait-for-app-state.sh local-manager HALT
+! stderr .
+exec -t 1m bash get-appinfo-status.sh app1
+! stdout 'HALTED'
+! stderr .
+
+# STEP 5: test reboot via controller bringing them back up
+# send reboot command and wait in background
+test eden.reboot.test -test.v -timewait=20m -reboot=1 -count=1 &
+exec -t 5m bash wait-for-app-state.sh local-manager RUNNING
+! stderr .
+exec -t 5m bash wait-for-app-state.sh app1 RUNNING
+! stderr .
+
+# STEP 6: Request for poweroff. Check that app1 stops before local_manager
+exec -t 1m bash put-devinfo-cmd.sh COMMAND_SHUTDOWN_POWEROFF &
+exec -t 5m bash wait-for-dev-state.sh POWERING_OFF
+! stderr .
+# Could it already have HALTED? Match HALTING or HALTED
+exec -t 5m bash wait-for-app-state.sh app1 HALT
+! stderr .
+exec -t 1m bash get-appinfo-status.sh local-manager
+! stdout 'RUNNING'
+! stderr .
+
+# Could it already have HALTED? Match HALTING or HALTED
+exec -t 5m bash wait-for-app-state.sh local-manager HALT
+! stderr .
+exec -t 1m bash get-appinfo-status.sh app1
+! stdout 'HALTED'
+! stderr .
+
+# STEP 7: Check qemu process is gone aka powered off
+exec sleep 120
+eden eve status
+! stdout process not running'
+
+# STEP 8: Restart EVE; check apps come up
+message 'Resetting of EVE'
+eden eve reset
```

Comment:
It should be `eden eve start`. Reset command will remove all apps from EVE configuration but will not affect qemu process state. In general seems we can run this only for qemu (see another comment).

Reply:
OK